### DISCUSSION TITLE
URL Spoofing

### DISCUSSION THREAD

**Discussion by akaranivia** [DSJAS/DSJAS#59] [General Q&A]
So I cant get the URL spoofing to work, I already did everything right in the host files
![image_2022-10-29_232554438](https://user-images.githubusercontent.com/55343047/198865646-7ccb51e9-d12e-4062-b6b3-729145e7babf.png)

I've also set it on the DSJAS admin page (valleybanknv.bank) and still this is what I get (tried other browsers)
![image](https://user-images.githubusercontent.com/55343047/198865688-774886bd-0ba1-4628-8f3b-03c7a7dd261e.png)

![image](https://user-images.githubusercontent.com/55343047/198865706-10dedf69-a94c-4e26-b83d-26815b0d1f35.png)

**Comment by SinisterSpatula** [comment]
That means you do not have a web server on your local machine or it isn't running, or something else is running and using the typical web ports (80/443) but is rejecting connections.  If dsjas is on a different machine you need to use that machines ip address instead of 127.x.x.x.
